ED/qa.html

```diff
@@ -617,15 +617,69 @@ <h3 property="schema:name">Test Report Description</h3>
-                    <li>Approved test reports MUST NOT include assertions of tests with rejected review status.</li>
-                    <li>The composition of the summary document is TBD.</li>
-                    <li>The report must also indicate the status of the publication as well as other visual cues.</li>
+                    <li>One <code>rdf:type</code> property whose object is <code>doap:Project</code>.</li>
+                    <li>Zero or one <code>doap:name</code> property to state the name of the project.</li>
+                    <li>Zero or one <code>doap:repository</code> property to indicate the location of of project's source code.</li>
+                    <li>Zero or one <code>doap:release</code> property to indicate the version information of a project release.</li>
+                    <li>Zero or one <code>doap:maintainer</code> property to refer to the maintainers of a project.</li>
                   </ul>
+
+                  <div class="note" id="test-report-description-doap" inlist="" rel="schema:hasPart" resource="#test-report-description-doap">
+                    <h4 property="schema:name"><span>Note</span>: Test Report Description: Description of a Project</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>The subject of the <code>doap:Project</code> for Test Report Description coincides with the object of the <code>earl:subject</code> in <a href="#test-assertion-description">Test Assertion Description</a>.</p>
+
+                      <p>To help distinguish project releases in test reports, it is encouraged to use versioned URIs for projects.</p>
+
+                      <p>While some information about the project can be accompanied with the test report, it is encouraged that projects are <a href="https://www.w3.org/2001/tag/doc/selfDescribingDocuments" rel="cito:citesAsInformation">self-describing documents</a>.</p>
+                    </div>
+                  </div>
+
+                  <p class="issue">What should be the recommendation for implementations that are not software projects? Perhaps equivalent to a top concept of <code>spec:ClassesOfProducts</code>?</p>
+
+                  <p>Test assertions:</p>
+
+                  <p>Test reports MUST incorporate additional information about test criterion indicating test authors and reviewers, review status, version of the test criterion, software and setup used to run the tests, provenance and coverage and test suite ( see also <a href="#test-assertion-description">Test Assertion Description</a>).</p>
+
+                  <p>Test reports with approved status MUST NOT include test assertions with rejected review status (<code>td:rejected</code>).</p>
```

Comment:
```suggestion
                  <p>Test reports with approved status MUST NOT include assertions related to test criteria with rejected review status (<code>td:rejected</code>).</p>
```
The assertion doesn't have a status, the test itself does.